conf/__init__.py

```diff
@@ -72,10 +72,6 @@
 
 test_web3_provider_list = [os.getenv("WEB3_PROVIDER")]
 
-# Liquid Tests
-liquid_api_key = os.getenv("LIQUID_API_KEY")
-liquid_secret_key = os.getenv("LIQUID_SECRET_KEY")
-
 # Wallet Tests
```

Comment:
You should not have to deal with other connectors

Reply:
That wasn't intentional. I have corrected it.